src/main/java/org/opensearch/ad/task/ADTaskCacheManager.java

```diff
@@ -81,6 +81,9 @@
     // Use this field to cache all HC tasks. Key is detector id
     private Map<String, ADHCBatchTaskCache> hcTaskCaches;
 
+    // This field is to cache all realtime tasks. Key is detector id
+    private Map<String, ADRealtimeTaskCache> realtimeTaskCaches;
```

Comment:
https://github.com/opensearch-project/anomaly-detection/pull/48#discussion_r636310719

@spbjss hi, Alex, this change answers your concern about why we don't name the cache manager as `HistoricalTaskCacheManager`